src/antlr4-runtime/scBaseVisitor.cpp

```diff
@@ -1,5 +1,5 @@
 
-// Generated from .\sc.g4 by ANTLR 4.9.2
+// Generated from /home/nhug/TaintAnalysisCpu/sc.g4 by ANTLR 4.8
```

Comment:
Shouldnt ANTLR 4.9.2 be used instead of 4.8?

Reply:
No clue why it's 4.8, since only ANTLR 4.9.2 is installed on my system